### PR TITLE
[Deployment] K8s deployment enhancement

### PR DESCRIPTION
**Pull request by yanmo96**
- Rather than wait certain time for k8s to be active, now the deployment script will check the status of k8s pod to make sure they are all up and running before continue. Thus can tolerate different deployment situations.
- Fixed some url links for ignite config files.

### CODE REVIEW

File: kubernetes/createYaml.sh [hunk 1]
```diff
@@ -28,32 +28,36 @@ if [[ "$OSTYPE" == "linux-gnu" ]]; then
...
   echo "Create yaml files done"
   
-  sleep 40s
-  
+  numberOfRnningPods="foo"
+  numberOfAllPods="bar"
+
+  while [ "$numberOfRnningPods" != "$numberOfAllPods" ];
+  do
+    sleep 10s
```
Comment (cj-chung) [inline kubernetes/createYaml.sh:36]: @yanmo96 Why we still need to wait 10s here?

**Comment by cj-chung** [comment]
@yanmo96 Please also add MIT license into these files.

File: kubernetes/createYaml.sh (hunk 1 above)
Comment (yanmo96) [reply kubernetes/createYaml.sh:36]: It checks the status every 10s

**Review by cj-chung — APPROVED** [review]
LGTM

**Pull request merged**